### PR TITLE
Homework for meeting 3

### PR DESCRIPTION
**Pull request by victor77dev**
This is the branch for homework

### CODE REVIEW

File: meeting_2/what_type_is_this.py [hunk 1]
```diff
@@ -3,8 +3,9 @@ def what_type_is_this(variable):
     what_type_is_this.line += 1
     print(f'line {what_type_is_this.line}: {variable} is ', end='')
```
Comment (victor77dev) [inline meeting_2/what_type_is_this.py:4]: ### Questions

1. Why there is `f` in front of the string?
2. Why we need `end=''` as the second argument?

Comment (victor77dev) [reply meeting_2/what_type_is_this.py:4]: Please put your answers here!

Comment (jjiajian) [reply meeting_2/what_type_is_this.py:4]: 1. f = format, equivalent to str.format()
2. end='' prevents a new line created after a print() statement

Comment (victor77dev) [reply meeting_2/what_type_is_this.py:4]: @jjiajian 
Note: you could use ` to quote the code like thing in markdown
[Here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a markdown cheat sheet from Github you could check.

**Pull request merged**